scripts/stackstorm.js

```diff
@@ -358,25 +368,26 @@ module.exports = function(robot) {
       }
       postDataHandler.postData(data);
 
-      res.send('{"status": "completed", "msg": "Message posted successfully"}');
-    } catch (e) {
-      robot.logger.error("Unable to decode JSON: " + e);
-      robot.logger.error(e.stack);
-      res.send('{"status": "failed", "msg": "An error occurred trying to post the message: ' + e + '"}');
+      res.send(JSON.stringify({
+        "status": "completed",
+        "msg": "Message posted successfully"
+      }));
+    } catch (err) {
+      logErrorAndExit(err, res)
```

Comment:
This is a bug in the original design, because `postDataHandler.postData(data);` sent data asynchronously (at least for Slack and MS Teams it does), so the catch block was simply never run. In the StackStorm web UI, it looked like the message was sent successfully, when it may not have been.

I have modified this to simply emit an error message letting the user know that the execution results may not be accurate. This is more in line with the previous code, but still probably isn't ideal.